docs-developer/deploying.md

```diff
@@ -44,3 +44,9 @@ When you're ready with a fix landed on `main`, you can push a new version to the
 
 You can find the Mozilla contacts about our deployment in [this Mozilla-only
 document](https://docs.google.com/document/d/16YRafdIbk4aFgu4EZjMEjX4F6jIcUJQsazW9AORNvfY/edit).
+
+# Deploying on a local nginx instance
+
+To deploy on nginx (without support for direct upload from the Firefox UI), run `yarn build-prod`
+and point nginx at the `dist` directory, which needs to be at the root of the webserver. Additionally,
+a `error_page 404 =200 /index.html;` directive needs to be added so that unknown URLs respond with index.html.
```

Comment:
Is it worth mentioning the headers we set using the `_headers` file? For a production server they're quite recommended, not so much when testing locally.